components/automate-chef-io/content/docs/iam-v2-overview.md

```diff
@@ -117,11 +117,12 @@ Ingest            |Chef-managed  |Ingest data into the system
 Compliance Viewer |Custom        |Viewer for compliance resources
 Compliance Editor |Custom        |Editor for compliance resources
 
-You can use these default Chef-created roles in your policies, or create your custom roles for more customized permissions.
+You can use these default Chef-created roles in your policies, or create more custom roles for further customized permissions.
 You can later edit any roles you create.
 
 Chef Automate also ships with policies that leverage these roles.
-Those Chef-managed policies are Viewers, Editors, Administrator, and Ingest. The custom ones are Compliance Viewers and Compliance Editors.
+Those Chef-managed policies are Viewers, Editors, Administrator, and Ingest.
```

Comment:
I realize these default **policies** aren't listed anywhere, but the roles are. Do the policy names match the role names? Asking because I'm not sure about the presence of all those plural `s`.

Reply:
The role is `Viewer` and the policy is `Viewers` and so on (aside from Administrator, the anomaly)